### PR TITLE
[16.0][ADD] stock_release_channel_partner_delivery_window

### PR DESCRIPTION
**Pull request by nguyenminhchien**
Depends on:

- [x] https://github.com/OCA/wms/pull/550
- [x] https://github.com/OCA/wms/pull/750
- [x] https://github.com/OCA/stock-logistics-workflow/pull/1378
- [x] https://github.com/OCA/stock-logistics-transport/pull/106

### CODE REVIEW

**Review by cyrilmanuel — APPROVED** [review]
not an expert but LGTM

**Review by simahawk — CHANGES_REQUESTED** [review]
the screenshot seems not relevant, right?

**Comment by nguyenminhchien** [comment]
> the screenshot seems not relevant, right?

removed, thanks.

File: stock_release_channel_partner_delivery_window/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+* Nguyen Minh Chien <chien@trobz.com>
```
Comment (simahawk) [inline stock_release_channel_partner_delivery_window/readme/CONTRIBUTORS.rst:1]: JE to be added here

File: stock_release_channel_partner_delivery_window/tests/test_release_window.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        picking.release_channel_id = False
+        picking.assign_release_channel()
+
+    @freeze_time("2023-09-01")
+    def test_assign_channel(self):
+        today = fields.Date.today()
+        self.channel.process_end_time = 0
+        self.channel.process_end_date = today + timedelta(days=1)  # 2023-09-02
+
+        # Anytime
```
Comment (simahawk) [inline stock_release_channel_partner_delivery_window/tests/test_release_window.py:67]: it would be better to see 1 test for each case

**Review by simahawk — COMMENTED** [review]
LG, just minor remarks

File: stock_release_channel_partner_delivery_window/readme/CONTRIBUTORS.rst (hunk 1 above)
Comment (nguyenminhchien) [reply stock_release_channel_partner_delivery_window/readme/CONTRIBUTORS.rst:1]: updated

File: stock_release_channel_partner_delivery_window/tests/test_release_window.py (hunk 2 above)
Comment (nguyenminhchien) [reply stock_release_channel_partner_delivery_window/tests/test_release_window.py:67]: added

**Review by jbaudoux — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Pull request merged**